src/core/server/path/index.ts

```diff
@@ -65,6 +65,8 @@ export type PathConfigType = TypeOf<typeof config.schema>;
 export const config = {
   path: 'path',
   schema: schema.object({
+    // getConfigPath returns the path until the kibana.yml config. We need to move 1 level up for the dir
+    config: schema.string({ defaultValue: () => resolve(getConfigPath(), '..') }),
```

Comment:
will it work with the absolute path? https://github.com/restrry/kibana/blob/edd8760bd428c925df375e129f82c46a7c7ba7f4/src/core/server/path/index.ts#L31
feel free to keep the file in the server folder. probably we can solve the problem later

Reply:
It will return the containing dir `'/etc/kibana'`. Why do you think it wouldn't work?

Reply:
what do consumers expect? file or directory? I'd assume path to a file https://github.com/elastic/kibana/pull/51478/files#diff-e46815170050691b2a3d13f171202117R52

Reply:
In the legacy world, they are running `dirname()` to the returned config path to retrieve the path. For the NP I thought it might make better sense to maintain consistency so both: `path.data` and `path.config` return the dirs for that (since plugins should not need to read anything from the `kibana.yml` config file). 
When migrating the telemetry plugin, they'll need to remove the usage of `dirname()`.
I hope it makes sense.

Reply:
Here is where telemetry uses that `config` path: https://github.com/elastic/kibana/blob/9e24cb268ed2e9a18e16584fb080e32a026d539d/src/legacy/core_plugins/telemetry/server/collectors/usage/telemetry_usage_collector.ts#L89

Reply:
ok. I hope they will realize that it's not a file path when migrating to the global config. 😄 Probably we should be a bit explicit and call them `configDir` and `dataDir`.

Reply:
Yeah! I like that. Way better than `path.dataPath` xD